java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSinkFactory.java

```diff
@@ -56,7 +56,15 @@ public void validate(
         Set<String> jdbcPks = new HashSet<>();
         Set<String> jdbcTableNames = new HashSet<>();
 
-        try (Connection conn = DriverManager.getConnection(jdbcUrl);
+        java.util.Properties info = new java.util.Properties();
+        if (config.getUser() != null) {
+            info.put("user", config.getUser());
+        }
+        if (config.getPassword() != null) {
+            info.put("password", config.getPassword());
+        }
+
+        try (Connection conn = DriverManager.getConnection(jdbcUrl, info);
```

Comment:
I think you can directly use `Connection getConnection(String url, String user, String password)` and it has null check inside.

Reply:
Yeah, that's my original code (as written in the PR description). Here I use `getConnection(jdbcUrl, info)` is to unify the function call in this `try (...)`,

Reply:
Don’t get it.😂 It seems `Connection conn = DriverManager.getConnection(jdbcUrl, config.getUser(), config.getPassword())`. correct me if I am wrong.

Reply:
Make sense. Fixed.